src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -18,6 +26,14 @@
 
 public class ModerationCommands {
 
+  private static final int REPORT_COOLDOWN_SECONDS = 15;
+
+  private static final Cache<UUID, Instant> lastReportSent =
+      CacheBuilder.newBuilder()
+          .weakKeys()
```

Comment:
Using uuis, no need for weak keys

Reply:
Ah thanks, I was unsure! I’ll adjust that now